x-pack/.gitignore

```diff
@@ -6,7 +6,6 @@
 /test/functional/apps/reporting/reports/session
 /plugins/reporting/.chromium/
 /plugins/reporting/.phantom/
-/.aws-config.json
```

Comment:
on one hand, it may be prudent to keep these around given input.  they haven't been in use since 4.6.  thoughts?

Reply:
Seems unnecessarily risky to me